### DISCUSSION TITLE
Using as an embedding api server

### DISCUSSION THREAD

**Discussion by zooteld** [Dicklesworthstone/swiss_army_llama#8] [Q&A]
is there an example of using swiss_army_llama or s.a.l. as a local embedding server setup? and how to call or post to it?

**Comment by Dicklesworthstone** [comment]
Yes it explains it in the readme and also in the swagger page.

**Comment by zooteld** [comment]
I must be missing it. what i was meaning is in my python app how can I send a request to the API?

**Comment by Dicklesworthstone** [comment]
Just use a library like httpx or requests. The swagger page gives the exact
curl commands for each endpoint. You can just ask ChatGPT to convert that
into python code using httpx. This is just basic REST api stuff, beginner
python. Good luck

**Reply by zooteld** [reply]
thank you, and yes I am a beginner lol
